.github/workflows/test-and-publish.yml

```diff
@@ -0,0 +1,47 @@
+# Github actions that run tests on each commit, and publish to PyPi on tagged releases
+# build using https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+
+name: Test and Release to PyPi
+
+on:
+  push
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install and build
+      run: >-
+        python -m
+        pip install ".[dev]"
+        build
+        --user
```

Comment:
Curious - Why do we need the `build` argument here? The original buildspec.yml just had "pip install .[dev]" https://github.com/aws/aws-jupyter-proxy/blob/master/buildspec.yml#L7-L8

Reply:
Fair, I was originally using build to explicitly compile the binaries and compose the tarball, but that's not needed. I can remove this step and the one beneath it, and replace them with just the install